completed react-form-controls - YoonHwan Cho

YoonHCho opened the pull request:
completed react-form-controls - YoonHwan Cho

![react-form-controls](https://user-images.githubusercontent.com/106636383/187999703-33195645-59eb-4783-8453-fd3125e20dc4.gif)

thebearingedge commented on react-form-controls/src/index.jsx:
```diff
@@ -0,0 +1,62 @@
+              onChange={ this.handlePasswordChange }
+            />
+          </label>
+          <label>
+          <button type="submit" onClick={ this.handleSubmit }>Sign Up</button>
```
You should listen for `onSubmit` of the form element instead of `onClick` of the button.

YoonHCho commented:
Hi Tim,
updated the according to the change request. `onSubmit `on the `form` element and removed `onClick` from `button` element

![image](https://user-images.githubusercontent.com/106636383/188033052-07df71ef-f0cf-448a-99c6-5d40983cf9f3.png)
